### PR TITLE
Add extra_help for TAG Review

### PR DESCRIPTION
**Pull request by dlaliberte**
This PR only adds extra_help for the TAG Review field of the I2S stage, based on Johanns text for the Privacy Sandbox.

image

(links open in new tab)

### CODE REVIEW

**Review by jrobbins — APPROVED** [review]

File: client-src/elements/form-field-specs.js [hunk 1]
```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
     label: 'TAG Review',
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:482]: Maybe:

```suggestion
    label: 'TAG Specification Review',
```

To distinguish this from the earlier TAG Early Design Review.

File: client-src/elements/form-field-specs.js [hunk 2]
```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
...
+    <p>
+      You will likely have asked for an "<a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=005-early-design-review.md"
+      >Early Design Review</a>" earlier in the process to get the TAG familiar with your feature. 
+      This isn't that.
+    </p>
+    <p>
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> at the I2E stage (right after is also fine).
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:503]:
```suggestion
      It's recommended that you file a TAG <a target="_blank" href=
        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
      >Specification Review</a> as soon as your specification is written, during your Origin Trial. 
```

File: client-src/elements/form-field-specs.js [hunk 3]
```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
     label: 'TAG Review',
     help_text: html`Link(s) to TAG review(s), or explanation why this is
                 not needed.`,
+    extra_help: html`
+    <p>
+      The <a target="_blank" href="https://www.w3.org/2001/tag/">W3C Technical Architecture Group</a> (TAG) 
+      is a special working group of the W3C that consists of a few appointed and elected members, 
+      all of whom are experienced members of the web standards community. 
+      The Blink launch process has a formal requirement for requesting a 
+      <a target="_blank" href="https://github.com/w3ctag/design-reviews">TAG review</a> for all features.
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:491]: @miketaylr mentioned that the launch process's formal requirement is only for the design review, not the specification review. Mike, could you check that the proposed text is what we should be telling feature authors?

File: client-src/elements/form-field-specs.js [hunk 4]
```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
...
+      and engagement from the TAG can take multiple weeks to multiple months.
+    </p>
+    <p>  
+      Note that only the request for TAG review is strictly required by the process. 
+      We don't block shipping on a response from the TAG, but a reasonable effort must be made 
+      to obtain their review with enough time for them to give feedback. 
+      There may be some work involved in preparing your feature for review (see the 
+        <a target="_blank" href=
+          "https://github.com/w3ctag/design-reviews/blob/main/.github/ISSUE_TEMPLATE/010-specification-review.md"
+        >submission template fields</a>).
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:514]: I think I'd move this chunk up to before the "A large number of Intents to Ship are delayed" sentence, and maybe put the paragraph break between them. It seems more connected to the "file the review" sentence than the "consequences of not filing the review".

File: client-src/elements/form-field-specs.js [hunk 5]
```diff
@@ -482,6 +482,37 @@ export const ALL_FIELDS = {
...
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> at the I2E stage (right after is also fine). 
+      A large number of Intents to Ship are delayed because a TAG review was only recently filed 
+      and engagement from the TAG can take multiple weeks to multiple months.
+    </p>
+    <p>  
+      Note that only the request for TAG review is strictly required by the process. 
+      We don't block shipping on a response from the TAG, but a reasonable effort must be made 
+      to obtain their review with enough time for them to give feedback.
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:510]: To simplify a bit:
```suggestion
      Note that the API owners can approve shipping even if the TAG hasn't replied to your review request, as long as you've made a reasonable effort
      to obtain their review with enough time for them to give feedback. 
```

**Review by jyasskin — COMMENTED** [review]
I'd like an API owner like @miketaylr to take a look at this, since we're trying to align our instructions with their process. Please take Mike's comments as more authoritative than mine.

File: client-src/elements/form-field-specs.js (hunk 1 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:482]: Yes, I was wondering about that.  I don't think we have any dependency on the label otherwise.

File: client-src/elements/form-field-specs.js (hunk 2 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:503]: I was going to ask what the I2E was.  Yet another name for Origin Trial, I guess.

File: client-src/elements/form-field-specs.js (hunk 5 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:510]: done

File: client-src/elements/form-field-specs.js (hunk 4 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js:514]: agreed

**Comment by dlaliberte** [comment]
After "Update...".

image

File: client-src/elements/form-field-specs.js (hunk 2 above)
Comment (jyasskin) [reply client-src/elements/form-field-specs.js]: I2E is "Intent to Experiment", which is the email the feature owner sends to blink-dev to get permission to run an Origin Trial. https://www.chromium.org/blink/launching-features/#step-5-optional-origin-trial describes this bit of the process.

I think of the I2E as an event (you press 'send' in your mail client) while "preparing for I2E" and "running an Origin Trial" are stages, since they last for a period of time. That might not align with what Chrome Status calls a "stage".

File: client-src/elements/form-field-specs.js [hunk 6]
```diff
@@ -479,9 +479,41 @@ export const ALL_FIELDS = {
...
+          "https://github.com/w3ctag/design-reviews/blob/main/.github/ISSUE_TEMPLATE/010-specification-review.md"
+        >submission template fields</a>).
+      A large number of Intents to Ship are delayed because a TAG review was only recently filed 
+      and engagement from the TAG can take multiple weeks to multiple months.
+    </p>
+    <p>  
+      Note that the API owners can approve shipping even if the TAG hasn't replied to your review request,
+      as long as you've made a reasonable effort
+      to obtain their review with enough time for them to give feedback.
+    </p>`,
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:516]: Paragraph nit:
```suggestion
    <p>
      It's recommended that you file a TAG <a target="_blank" href=
        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
      >Specification Review</a> as soon as your specification is written, during your Origin Trial. 
      There may be some work involved in preparing your feature for review (see the 
        <a target="_blank" href=
          "https://github.com/w3ctag/design-reviews/blob/main/.github/ISSUE_TEMPLATE/010-specification-review.md"
        >submission template fields</a>).
    </p>
    <p>  
      A large number of Intents to Ship are delayed because a TAG review was only recently filed 
      and engagement from the TAG can take multiple weeks to multiple months.
      Note that the API owners can approve shipping even if the TAG hasn't replied to your review request,
      as long as you've made a reasonable effort
      to obtain their review with enough time for them to give feedback.
    </p>`,
```

Comment (dlaliberte) [reply client-src/elements/form-field-specs.js]: done

File: client-src/elements/form-field-specs.js (hunk 3 above)
Comment (miketaylr) [reply client-src/elements/form-field-specs.js:491]: Let me chat with some additional API OWNERS in our weekly meeting tomorrow - my read on https://www.chromium.org/blink/launching-features/ is that we require design reviews before shipping (generally), not design reviews + specification reviews.

**Comment by dlaliberte** [comment]
Latest version:

image

File: client-src/elements/form-field-specs.js (hunk 3 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js]: Mike posted about this at:   https://groups.google.com/a/chromium.org/g/blink-api-owners-discuss/c/py_scEqzKPE
So the answer in replies (so far) is still unknown, and it seems possible that only a design review is required.

Comment (miketaylr) [reply client-src/elements/form-field-specs.js]: Once we have consensus, I'll reply here (and then go clarify the requirements).

File: client-src/elements/form-field-specs.js [hunk 7]
```diff
@@ -479,17 +479,48 @@ export const ALL_FIELDS = {
...
+    <p>
+      You will likely have asked for an "<a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=005-early-design-review.md"
+      >Early Design Review</a>" earlier in the process to get the TAG familiar with your feature.
+      This isn't that.
+    </p>
+    <p>
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> as soon as your specification is written, during your Origin Trial.
```
Comment (miketaylr) [inline client-src/elements/form-field-specs.js:503]: Note that the Origin Trials stage won't apply to all new features with specs. We could possibly say "It's recommended that you file a TAG Specification Review as soon as your specification is written, and at least a month ahead of sending an Intent to Ship."

Or similar. But it's probably fine as-is (but might confuse some folks. 🤷)

**Review by miketaylr — APPROVED** [review]
LGTM - we met as API OWNERS and aligned on basically the text here.

File: client-src/elements/form-field-specs.js (hunk 7 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js]: Updated with your suggestions:
```
It's recommended that you file a TAG Specification Review as soon as your specification is written, 
and at least a month ahead of sending an Intent to Ship.  
There might be enough time if you file for review of your specification during your Origin Trial, assuming your feature requires one.
```

Maybe this is a bit wordy and unnecessary.

File: client-src/elements/form-field-specs.js [hunk 8]
```diff
@@ -479,17 +479,48 @@ export const ALL_FIELDS = {
...
+    <p>
+      You will likely have asked for an "<a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=005-early-design-review.md"
+      >Early Design Review</a>" earlier in the process to get the TAG familiar with your feature.
+      This isn't that.
+    </p>
+    <p>
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> as soon as your specification is written, and at least a month ahead of sending an Intent to Ship.  There might be enough time if you file for review of your specification during your Origin Trial, assuming your feature requires one.
```
Comment (jyasskin) [inline client-src/elements/form-field-specs.js:503]:
```suggestion
      >Specification Review</a> as soon as your specification is written, and at least a month ahead of sending an Intent to Ship.
```

**Review by jyasskin — APPROVED** [review]

File: client-src/elements/form-field-specs.js (hunk 8 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js]: done

File: client-src/elements/form-field-specs.js (hunk 7 above)
Comment (dlaliberte) [reply client-src/elements/form-field-specs.js]: removed the Origin Trial part.
